pkg/apis/admissionregistration/validation/validation.go

```diff
@@ -390,6 +395,13 @@ var supportedReinvocationPolicies = sets.NewString(
 	string(admissionregistration.IfNeededReinvocationPolicy),
 )
 
+var supportedValidationPolicyReason = sets.NewString(
+	string(metav1.StatusReasonUnauthorized),
```

Comment:
While working through the display and client impact of the various returned reasons in https://github.com/kubernetes/kubernetes/pull/113314#discussion_r1014869210, I realized that returning unauthorized errors (401) can kick clients into credential renew codepaths, which we *don't* want to do, so drop `StatusReasonUnauthorized`.

I'm pretty happy with the remaining reasons (forbidden, invalid, toolarge)